scripts/ci/check-stylish-network.sh

```diff
@@ -2,13 +2,13 @@
 
 set -euo pipefail
 
+export LC_ALL=C.UTF-8
 fd -p io-sim -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
 fd -p io-classes -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
 fd -p strict-stm -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
 fd -p typed-protocols -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
 # TODO CPP pragmas in export lists are not supported by stylish-haskell
-fd -p network-mux -e hs -E Setup.hs -E network-mux/src/Network/Mux/Bearer/Pipe.hs -E network-mux/src/Network/Mux/Channel.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
-fd -p ouroboros-network* -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
+fd -p network-mux -e hs -E Setup.hs -E network-mux/src/Network/Mux/TCPInfo.hs -E network-mux/src/Network/Mux/Bearer/Pipe.hs -E network-mux/src/Network/Mux/Channel.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
+fd -p ouroboros-network-framework -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
+fd . './ouroboros-network' -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
 fd -p cardano-client -e hs -E Setup.hs -X stylish-haskell -c .stylish-haskell-network.yaml -i
-
-git diff --exit-code
```

Comment:
Was this line removed intentionally (from both scripts)? What problem was it causing?

Reply:
OK. And both scripts run `set -e`, so if any `check-stylish` invocation fails, we'll get a helpful non-0 exit code from the script. So that aspect of `git diff` hasn't been lost. (I'm assuming `fd` propagates the invoked exe's exit code.)